tests/test_client_session.py

```diff
@@ -712,6 +712,12 @@ async def test_client_session_timeout_argument() -> None:
     assert session.timeout == 500
 
 
+async def test_client_session_timeout_zero() -> None:
+    timeout = client.ClientTimeout(total=10, connect=0, sock_connect=0, sock_read=0)
+    async with ClientSession(timeout=timeout) as session:
+        await session.get("http://example.com")
```

Comment:
I'd add some assertion because without an `assert` it's totally unclear what side-effect is being checked.